pkg/internal/client/client.go

```diff
@@ -21,16 +21,22 @@ package client
 
 import (
 	"context"
+	"log"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8scorev1 "k8s.io/api/core/v1"
+	k8smetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/rest"
 
+	netattdefv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
```

Comment:
nit: this was previously defined as "networkv1"

Reply:
DONE